### DISCUSSION TITLE
Adding Dark Mode to Decap CMS

### DISCUSSION THREAD

**Discussion by privatemaker** [decaporg/decap-cms#7084] [Show and tell]
I'm just posting something mentioned in Discord (for those who are not there) which is useful and was created by @micvolo 🙏🏻

- Decap CMS dark-mode CSS: [dark.css](https://stra.studio/cms/dark.css)

**Comment by micvolo** [comment]
credits to [this repo](https://github.com/Nats-ji/netlify-cms-dark-mode)!

I just added some new classes for the new updates & integrated a better mobile responsivness.

Every time there is a new update i have to manually add some classes, i try to keep it up to date.

thanks for the mention btw!
